src/DataProtection/Cryptography.KeyDerivation/src/Microsoft.AspNetCore.Cryptography.KeyDerivation.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core utilities for key derivation.</Description>
-    <TargetFrameworks>netstandard2.0;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
Is it possible to remove `netcoreapp2.0` from this? I don't remember the context: Why target the older TFM?

Reply:
Would that be considered a breaking change? I don't know if that would be able to make a change such as modifying a TFM in servicing but we have said that adding an additional TFM is allowed in the past.

Reply:
I unfortunatley agree removing 'netcoreapp2.0' would be considered a breaking change. But, I'd still like to understand why the project targets the older TFM in the first place.

Reply:
My guess would be that we just overlooked it? We don't work with this package often. It is strange that we didn't automatically update the TFM to the latest though.

Maybe we should follow up in 5.0 to remove the previous TFM.

Reply:
Sounds reasonable. Please file a follow-up issue or open a PR proactively.